docs/current/cli/389936-get-started.md

```diff
@@ -0,0 +1,3 @@
+---
+slug: /cli/get-started
+---
```

Comment:
Can you please provide an example for this get started?

Reply:
> Can you please provide an example for this get started?

Sure, I'll send some examples today about the things you can do in the CLI